locust/runners.py

```diff
@@ -760,7 +778,7 @@ def _wait_for_workers_report_after_ramp_up(self) -> float:
 
         assert False, "not supposed to reach that"
 
-    def stop(self):
+    def stop(self, send_stop_to_client: bool = True):
```

Comment:
client is synonymous with worker, right? We dont have to change that in this PR, but I think we should clean this up in the future and user "worker" everywhere.

Reply:
Yes, I tried to use `client` in the `Runner` classes because it was the nomenclature already used, but workers would make more sense.